packages/@okta/i18n/package.json

```diff
@@ -1,106 +1,80 @@
 {
   "_args": [
     [
-      "@okta/i18n@42.1.0",
-      "/Users/mauricio.castillosilva/okta/okta-signin-widget"
+      {
```

Comment:
Not sure about this change. @mauriciocastillosilva-okta ?

Reply:
I've seen this with a simple string array or an object array, but I have seen no difference on how this affects...
My guess is that this happens because of different versions of npm

Reply:
Talked to Mau offline. This is ok 👍